### ISSUE TITLE
NERDTree master opening an existing buffer jumps to different tab

### ISSUE THREAD

**Issue by agibralter** [scrooloose/nerdtree#170]
I've noticed that the behavior between 4.2.0 and the current master (https://github.com/scrooloose/nerdtree/compare/4.2.0...2cb0fc78fb) changes the behavior of open: if a buffer is open in a different tab, hitting enter on a nerdtree node will jump to the tab with that buffer. Is there anyway to get the 4.2.0 behavior of opening the buffer in the last active window of the current tab?

**Comment by blueyed** [comment]
Seems to have been addressed in 87b2780.

**Comment by adgoudz** [comment]
It wasn't clear to me how to leverage the fix in 87b2780. It looks like I need to override the mapping for `g:NERDTreeMapActivateNode` but ideally I should be able to set "reuse" via an option.

For now, adding this to `.vim/nerdtree_plugin/overrides.py` works for me:
```                                                                                                                             
function s:SID()                                                                                                                  
    return matchstr(expand('<sfile>'), '<SNR>\zs\d\+\ze_SID$')                                                                    
endfun                                                                                                                            
                                                                                                                                  
let s = '<SNR>' . s:SID() . '_'                                                                                                   
                                                                                                                                  
" Mapping Overrides                                                                                                               
                                                                                                                                  
call NERDTreeAddKeyMap({ 'key': g:NERDTreeMapActivateNode, 'scope': 'FileNode', 'callback': s.'activateFileNode', 'override': 1 })
                                                                                                                                  
" Mapping Functions                                                                                                               
                                                                                                                                  
function! s:activateFileNode(node)                                                                                                
    call a:node.activate({'reuse': 'currenttab', 'where': 'p'})                                                                   
endfunction                                                                                                                       
```

**Comment by vpsx** [comment]
@adgoudz This worked for me:
```
let NERDTreeCustomOpenArgs = {'file': {'reuse': 'currenttab', 'where': 'p'}, 'dir': {}}
```
(and then use custom open `<CR>`, or do e.g. `let NERDTreeMapCustomOpen = 'o'`)

Sorry, not sure if it's bad etiquette to comment on a closed issue after a billion years, but I googled myself here and this would have helped me 😅
